app/views/products/show.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= render 'admin/products/show'%>
```

Comment:
Now I see why partial is created. But I would suggest we usually have more / different functionalities for admin. In shared view, we will have to add conditionals. So to avoid that, keep the views separated.  No problem if HTML code is duplicated.